src/cogs/admin_commands.py

```diff
@@ -126,3 +127,48 @@ async def create_channel_teams(self, ctx, channel_id, number_of_teams=2):
             )
 
         await ctx.channel.send(embed=teams_embed)
+
+    @commands.command(name="send-survey")
+    @commands.has_role(ADMIN_ROLE_ID)
+    async def send_survey(self, ctx, survey_id):
+        """
+        Only member with admin role can run this command
+        Command sends survey (id of survey in first parameter) to all users who are podcast-fans
+        (They answered in welcome quiz that they are ok with sending more surveys)
+        """
+
+        logging.info(f"Executing send-survey command for survey_id {survey_id}")
+
+        if db.check_if_survey_exists(survey_id):
+            survey_fans = db.find_all_by_alias("survey-fan")
+            users_who_started_survey = db.get_all_user_ids_from_survey_progress(
+                survey_id
+            )
+            final_users_to_send_survey_to = list(
+                set(survey_fans) - set(users_who_started_survey)
+            )
+
+            sent_to = []
+            for user_id in final_users_to_send_survey_to:
+                member = get_member(self.bot, user_id)
+
+                channel_name = f"{member.display_name}-soc-dotaznik"
```

Comment:
So we want to reuse the channels? I think we should discuss this offline.

Reply:
No, actually new channels are created for each survey

Reply:
With the same name?